CrisisTextLine/Elmer
# Update Canvas grade for users attending first shift

#### What's this PR do?
Create a route for the platform to update canvas when a user attends their shift
#### Where should the reviewer start?
#### How should this be manually tested?
Change the name of the assignment in the canvas index route to one that currently exists in canvas. (or you can use assaf@crisistextline.org, assignment 'Checkpoint #17: Attend First Shift' I'll show you my account).
Postman post to /canvas/firstShift with a body that contains users: [{name: " ", email: "}] and see if the user with that email address had their assignment updated
#### Any background context you want to provide?
#### What are the relevant tickets?
https://admin.crisistextline.org/jira/secure/RapidBoard.jspa?rapidView=5&view=detail&selectedIssue=INT-236
#### Questions:
Should the functionality in the router be in a different location? Maybe a factory or moved into updateCanvas?
